code/code/spec/spec_mobs.cc

```diff
@@ -4560,6 +4559,662 @@ int engraver(TBeing *ch, cmdTypeT cmd, const char *arg, TMonster *me, TObj *o)
   return FALSE;
 }
 
+// Resize and engrave for worn items
+int tailor(TBeing *ch, cmdTypeT cmd, const char *arg, TMonster *me, TObj *o)
```

Comment:
Why do you say it's not used outside this file?

Reply:
You're calling it lower down in the same file. You're not calling it from anywhere else.
It's a new function, therefore it can't be called in another place, in old code. Qed.

Reply:
I guess technically it's called from here: https://github.com/sneezymud/sneezymud/blob/master/code/code/spec/spec_mobs.cc#L91-L115 the same as all the other mob procs. With my limited experience I assume this could probably all be wrangled to work by putting the functions in a different namespace. But seeing as you commented on a single proc I wasn't sure if you meant we should refactor all procs to be in an anonymous namespace or just this one.

I guess my question was more like...

Do you mean we should do this for all procs? Or at least the ones in this file? or is there something special about this proc? I just don't quite understand the context around why that should be necessary for one of the procs I added but not all the other ones and was hoping you could provide more info so that I know why I'm doing it.